lib/execution_engine2/execution_engine2Impl.py

```diff
@@ -301,50 +292,52 @@ def get_job_logs(self, ctx, params):
            loaded lines next time.) -> structure: parameter "lines" of list
            of type "LogLine" -> structure: parameter "line" of String,
            parameter "is_error" of type "boolean" (@range [0,1]), parameter
-           "last_line_number" of Long
+           "ts" of String, parameter "last_line_number" of Long
         """
         # ctx is the context object
         # return variables are: returnVal
-        # BEGIN get_job_logs
+        #BEGIN get_job_logs
         mr = SDKMethodRunner(self.config)
         returnVal = mr.view_job_logs(
             job_id=params["job_id"], skip_lines=params.get("skip_lines", None), ctx=ctx
         )
-
-        # BEGIN add_job_logs
-
-        # END get_job_logs
+        #END get_job_logs
 
         # At some point might do deeper type checking...
         if not isinstance(returnVal, dict):
-            raise ValueError(
-                "Method get_job_logs return value "
-                + "returnVal is not type dict as required."
-            )
+            raise ValueError('Method get_job_logs return value ' +
+                             'returnVal is not type dict as required.')
         # return the results
         return [returnVal]
 
-    def finish_job(self, ctx, job_id, params):
+    def finish_job(self, ctx, params):
         """
         Register results of already started job
-        :param job_id: instance of type "job_id" (A job id.)
-        :param params: instance of type "FinishJobParams" (Either 'result',
-           'error' or 'is_canceled' field should be defined; result - keeps
-           exact copy of what original server method puts in result block of
-           JSON RPC response; error - keeps exact copy of what original
-           server method puts in error block of JSON RPC response;
-           is_cancelled - Deprecated (field is kept for backward
-           compatibility), please use 'is_canceled' instead.) -> structure:
-           parameter "result" of unspecified object, parameter "error" of
-           type "JsonRpcError" (Error block of JSON RPC response) ->
-           structure: parameter "name" of String, parameter "code" of Long,
-           parameter "message" of String, parameter "error" of String,
-           parameter "is_cancelled" of type "boolean" (@range [0,1]),
-           parameter "is_canceled" of type "boolean" (@range [0,1])
+        :param params: instance of type "FinishJobParams" (error_message:
+           optional if job is finished with error) -> structure: parameter
+           "job_id" of type "job_id" (A job id.), parameter "error_message"
+           of String
         """
         # ctx is the context object
-        # BEGIN finish_job
-        # END finish_job
+        #BEGIN finish_job
+        mr = SDKMethodRunner(self.config)
+        mr.finish_job(params.get('job_id'), error_message=params.get('error_message'))
```

Comment:
This should default to empty string or None if not supplied

Reply:
sure. But params.get defaults to `None` if `error_message` is not found.

Reply:
You are right, my mistake.